src/wifi/model/mac-low.h

```diff
@@ -1147,11 +1147,14 @@ double mpduSnr;
     Ptr<WifiMacQueue> aggregateQueue;
   } AllocationParameters;
 
-  typedef std::map<AllocationID, AllocationParameters> AllocationPeriodsTable;
+  typedef std::pair<Mac48Address, Mac48Address> AddressPair;
+  typedef std::map<AddressPair, AllocationParameters> AllocationPeriodsTable;
   typedef AllocationPeriodsTable::const_iterator AllocationPeriodsTableCI;
   typedef AllocationPeriodsTable::iterator AllocationPeriodsTableI;
   AllocationPeriodsTable m_allocationPeriodsTable;
```

Comment:
it would be nice to add some documentation on this variable as it seems quite important and not straightforward to understand without looking at a bunch of code first